PowerEditor/src/Parameters.cpp

```diff
@@ -1991,120 +1992,110 @@ void NppParameters::updateFromModelXml(TiXmlNode* rootUser, ConfXml whichConf)
 	std::wstring modelXmlPath(_nppPath);
 	pathAppend(modelXmlPath, modelXmlFilename);
 
-	// compare the *.model.xml's filesystem "modified" timestamp to the value stored in the user file modelModifTimestamp attribute
-	const wchar_t* wc_user_modelModifTimestamp = peRootUser->Attribute(L"modelModifTimestamp");
-	std::wstring ws_user_modelModifTimestamp = wc_user_modelModifTimestamp ? wc_user_modelModifTimestamp : L"";
+	// compare the *.model.xml's filesystem "modified" timestamp (date only) to the value stored in the user file modelLastModifDate attribute
+	int userModelModifDate = 0;
+	peRootUser->Attribute(L"modelLastModifDate", &userModelModifDate);

```

Comment:
I saw that in the comments https://github.com/notepad-plus-plus/notepad-plus-plus/commit/723917722644afdd0ec75d5dad5af51ffe419b78#r172928069, there is still some discussion about what the name should be.

Whatever name you settle on, it needs to go in the theme.xsd and langs.xsd:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/ecb3a141260e0ae7c20ddc0ae627d1fd28ed2b40/PowerEditor/Test/xmlValidator/theme.xsd#L101

and

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/ecb3a141260e0ae7c20ddc0ae627d1fd28ed2b40/PowerEditor/Test/xmlValidator/langs.xsd#L58